tests/unit/src/main/scala/tests/BaseWorksheetLspSuite.scala

```diff
@@ -297,26 +297,31 @@ abstract class BaseWorksheetLspSuite(scalaVersion: String)
     } yield ()
   }
 
-  testAsync("definition") {
-    for {
-      _ <- server.initialize(
-        s"""
-           |/metals.json
-           |{"a": {"scalaVersion": "$scalaVersion"}}
-           |/a/src/main/scala/Main.worksheet.sc
-           |val message = "Hello World!"
-           |println(message)
-           |""".stripMargin
-      )
-      _ <- server.didOpen("a/src/main/scala/Main.worksheet.sc")
-      _ = assertNoDiff(
-        server.workspaceDefinitions,
-        """|/a/src/main/scala/Main.worksheet.sc
-           |val message/*L0*/ = "Hello World!"
-           |println/*Predef.scala*/(message/*L0*/)
-           |""".stripMargin
-      )
-    } yield ()
-  }
+  // NOTE(olafur) this test fails unpredicatly on Windows with
```

Comment:
I think it's because the filesystem might be slower on Windows - it seems like a race. I will need to do another "Windows session" and try to fix the rest of the test.